src/helpers/xml-builder.js

```diff
@@ -2,7 +2,6 @@
 /* eslint-disable no-case-declarations */
 /* eslint-disable no-plusplus */
 /* eslint-disable no-else-return */
-/* eslint-disable no-unused-vars */
```

Comment:
I removed this as I am not in favour of blanket file lint rules - specially the `no-unused-vars`.

Reply:
sounds good.